app/controllers/users/add_user.js

```diff
@@ -0,0 +1,148 @@
+import config from "stock/config/environment";
+import AsyncMixin, { ERROR_STRATEGIES } from "stock/mixins/async";
+import ImageUploadMixin from "stock/mixins/image_upload";
+import AjaxPromise from "stock/utils/ajax-promise";
+import Ember from "ember";
+
+export default Ember.Controller.extend(AsyncMixin, ImageUploadMixin, {
+  newUploadedImage: null,
+  i18n: Ember.inject.service(),
+  messageBox: Ember.inject.service(),
+
+  titles: Ember.computed(function() {
+    let translation = this.get("i18n");
+    let mr = translation.t("order.user_title.mr");
+    let mrs = translation.t("order.user_title.mrs");
+    let miss = translation.t("order.user_title.miss");
+    let ms = translation.t("order.user_title.ms");
+
+    return [
+      { name: mr, id: "Mr" },
+      { name: mrs, id: "Mrs" },
+      { name: miss, id: "Miss" },
+      { name: ms, id: "Ms" }
+    ];
+  }),
+
+  setEmailorMobile: Ember.computed("email", "mobilePhone", function() {
+    const email = this.get("email");
+    const mobile = this.get("mobilePhone");
+
+    if (/^[456789]\d{7}/.test(mobile) || /^[^@\s]+@[^@\s]+/.test(email)) {
+      return false;
```

Comment:
by reading the method name, this should set either email or mobile, but its actually setting boolean value whose value is used in required field.
This method name needs to be changed